actions/apt_action.go

```diff
@@ -20,10 +21,15 @@ Optional properties:
 - recommends -- boolean indicating if suggested packages will be installed
 
 - unauthenticated -- boolean indicating if unauthenticated packages can be installed
+- local-packages-dir -- directory containing local packages to be installed
```

Comment:
it's not entirely clear without thinking about it where this directory lives - could make it clear that it is under `$RECIPEDIR`